src/SFA.DAS.EmployerFinance.AcceptanceTests/Extensions/ObjectContainerExtensions.cs

```diff
@@ -46,7 +46,7 @@ public static void RegisterMocks(this IObjectContainer objectContainer, IContain
             objectContainer.RegisterMock<ICurrentDateTime>(container);
             objectContainer.RegisterMock<IEmployerAccountRepository>(container);
             objectContainer.RegisterMock<IEventsApi>(container);
-            objectContainer.RegisterMock<IMembershipRepository>(container);
+            //objectContainer.RegisterMock<IMembershipRepository>(container);
```

Comment:
use it or lose it

Reply:
Sorry that one was one me; temp. hack to get it building until I could fix up mock Auth properly, but missed cleaning that up. good spot, sorted.